backend/prompt_studio/prompt_studio_core_v2/prompt_studio_helper.py

```diff
@@ -1157,7 +1157,11 @@ def _fetch_single_pass_response(
         tool_settings[TSPKeys.ENABLE_CHALLENGE] = tool.enable_challenge
         tool_settings[TSPKeys.ENABLE_HIGHLIGHT] = tool.enable_highlight
         tool_settings[TSPKeys.CHALLENGE_LLM] = challenge_llm
-
+        tool_settings[TSPKeys.ENABLE_HIGHLIGHT] = tool.enable_highlight
+        tool_settings[TSPKeys.PLATFORM_POSTAMBLE] = getattr(
```

Comment:
@Deepak-Kesavan Should these keys be added in prompt studio registry for export?

Reply:
These keys will be already there in tool export.